apps/bfd-model/bfd-model-rif-samples/src/main/java/gov/cms/bfd/model/rif/samples/StaticRifResource.java

```diff
@@ -593,7 +640,7 @@ public static void main(String[] args) throws Exception {
         parser.forEach(
             r -> {
               if (resource.getRifFileType().getIdColumn() != null)
-                uniqueIds.add(r.get(resource.getRifFileType().getIdColumn()));
+                uniqueIds.add(r.get(resource.getRifFileType().getIdColumn().get()));
```

Comment:
we cannot call get on an Optional without first checking if it is present and handling the empty case appropriately